en/mongodb.markdown

```diff
@@ -45,16 +45,15 @@ This does bring up the first thing you should know about MongoDB: its drivers. M
 
 As you read through this, I encourage you to play with MongoDB to replicate what I demonstrate as well as to explore questions that you might come up with on your own. It's easy to get up and running with MongoDB, so let's take a few minutes now to set things up.  You'll need to have a MongoDB server running somewhere, as well as a MongoDB client (CLI or GUI) running locally.
 
-For the client part, install either MongoDB Shell from [the official page](https://www.mongodb.com/docs/mongodb-shell/) or if you prefer GUI to command line shell, you can use [MongoDB Compass](https://www.mongodb.com/try/download/compass), an open source GUI for MongoDB. MongoDB Compass has the shell built-in so you can switch between using GUI and command line as you wish.
+For the client part, install either the MongoDB Shell from [the official page](https://www.mongodb.com/try/download/shell) or if you prefer GUI to command line shell, you can use the [MongoDB Compass](https://www.mongodb.com/try/download/compass), an open source GUI for MongoDB. MongoDB Compass has the shell built-in so you can switch between using GUI and command line as you wish.
```

Comment:
why?  Installation page shows how they can do it with `brew` on Mac and yum/apt-get on Linux, directing them straight to "download this" means they may do unnecessary work when maybe they don't need to...